platform/mkrules/src/sonic-sairedis/debian/autoreconf.before

```diff
@@ -2335,725 +2329,6 @@ e63e352f5fd91d7a06dc7ed3f57af746  ./SAI/meta/saisanitycheck.c
-016b4e317bd2ac26b67f5bd57c51627c  ./SAI/bm/behavioral-model/src/bm_sim/dev_mgr_bmi.cpp
-f215edd156e0ab70bef962bb708ee9c6  ./SAI/bm/behavioral-model/src/bm_sim/ageing.cpp
-7325eef20e017e12878f43e66f4ab259  ./SAI/bm/behavioral-model/src/bm_sim/P4Objects.cpp
-6f7c22001c214f177ff0fab9f0cd1c78  ./SAI/bm/behavioral-model/src/bm_sim/lookup_structures.cpp
-ac4290d8f42953b07dff63728d68b479  ./SAI/bm/behavioral-model/src/bm_sim/conditionals.cpp
-7608f6f08119fee71983a4ff56d999aa  ./SAI/bm/behavioral-model/src/bm_sim/md5.h
-14f9a26bfb2f2dfa29fa902ffdfb7433  ./SAI/bm/behavioral-model/src/bm_sim/checksums.cpp
-e69eafb1b7d7c42b2af8c78f7196fde1  ./SAI/bm/behavioral-model/src/bm_sim/pipeline.cpp
-3ac92230abc77927e7e1c55b13d4be67  ./SAI/bm/behavioral-model/src/bm_sim/xxhash.h
-1e6903c47d9a076004f2ac63dd36cb00  ./SAI/bm/behavioral-model/src/bm_sim/options_parse.cpp
-5fbb3869039b761963f2457ddced1294  ./SAI/bm/behavioral-model/src/bm_sim/match_tables.cpp
-5637e213d1b81fe5d96d013c07b5cb15  ./SAI/bm/behavioral-model/src/bm_sim/crc_tables.h
-8a3d5f792bb41ea5b1c64633ce4c8583  ./SAI/bm/behavioral-model/src/bm_sim/bytecontainer.cpp
-8f6e76dca49ad41a433da9b5643e8f9a  ./SAI/bm/behavioral-model/src/bm_sim/simple_pre_lag.cpp
-b907f12557bfe3e356656cae1c79aad7  ./SAI/bm/behavioral-model/src/bm_sim/transport.cpp
-38a19fa07ffc7eec3ccb97ec24a2f1e7  ./SAI/bm/behavioral-model/src/bm_sim/logger.cpp
-6a01ecae65076cbdce168b3889bca144  ./SAI/bm/behavioral-model/src/bm_sim/packet.cpp
-9b1d92c85ec5147807207423671c4849  ./SAI/bm/behavioral-model/src/bm_sim/pcap_file.cpp
-99545e5d45e81c93da731114263de33e  ./SAI/bm/behavioral-model/src/bm_sim/lpm_trie.h
-0022b7329cd28556d712c7f151c67d64  ./SAI/bm/behavioral-model/src/bm_sim/extract.h
-d5f14d1c63e43e7c5a3a7e9f20cf5e31  ./SAI/bm/behavioral-model/src/bm_sim/.gitignore
-e1542afbc7d698c6761af80d7c016146  ./SAI/bm/behavioral-model/src/bm_sim/target_parser.cpp
-95f7bb49a32c727c9045a469ed8a7192  ./SAI/bm/behavioral-model/src/bm_sim/stateful.cpp
-6e8a8d6cd232c755886e895a8f9c8152  ./SAI/bm/behavioral-model/src/bm_sim/actions.cpp
-2d8340629a3856599702b19b5f5cb31c  ./SAI/bm/behavioral-model/src/bm_sim/headers.cpp
-3b77231f7e9003e7aba286e93dda2e1f  ./SAI/bm/behavioral-model/src/bm_sim/calculations.cpp
-c0025624abf26a13d6943bb2ff929e3a  ./SAI/bm/behavioral-model/src/bm_sim/meters.cpp
-4a47be56ab5139edc128611f24d3928a  ./SAI/bm/behavioral-model/src/bm_sim/context.cpp
-87d4c1ebf7037f54d2d710f750727c9b  ./SAI/bm/behavioral-model/src/bm_sim/port_monitor.cpp
-d4f88b12fb455b291d6ffc4dd42356f6  ./SAI/bm/behavioral-model/src/bm_sim/event_logger.cpp
-4a9d65c0d8e76965701552b5ae69ba96  ./SAI/bm/behavioral-model/src/bm_sim/transport_nn.cpp
-8273ae24f33696a3e632cea51bcb6a73  ./SAI/bm/behavioral-model/src/bm_sim/utils.h
-21aa5c8efe1f1025123151038309f566  ./SAI/bm/behavioral-model/src/bm_sim/enums.cpp
-7d38cd4358db266a3a19544b74f82ee8  ./SAI/bm/behavioral-model/src/bm_sim/action_profile.cpp
-c2de4171f8b65783dcc136f385a71998  ./SAI/bm/build_p4_target.sh
```

Comment:
is removing reference to all bm files the right thing? I thought it was available in master.

Reply:
I didn't find autoreconf.before/after in upstream sonic-sairedis. And not sure why they were needed. I removed them and build/test are still successful. I will remove them from the commit.